### ISSUE TITLE
Languages module Alt text and Width Height of Images

### ISSUE THREAD

**Issue by niosme** [joomla/joomla-cms#46743]
### Steps to reproduce the issue
1. Images dont have and alt text always as it should have and also no width and height specified. This occurs pagespeed errors that could be fixed with ovewrite, but i think its nesecarry to be added in the default module for better accessibility. Even if the param full_name is turned on doesnt fill the alt text.

Also size of the image on better practices is 18x12 and pagespeed suggests a bigger image like 27x18 for the image to be as default size.

<img width="957" height="727" alt="Image" src="https://github.com/user-attachments/assets/7fac9a43-7df4-47f2-8ec2-60848217abd8" />

In languages module images of languages are missing the alt text and the width and height variables.

### Expected result

`<img title="English" width="18" height="12" src="/media/mod_languages/images/en_gb.gif" alt="English">`

### Actual result

`<img src="/media/mod_languages/images/en_gb.gif">`
or
`<img title="English" src="/media/mod_languages/images/en_gb.gif">`

### Fix
On every occurence of

`<?php echo HTMLHelper::_('image', 'mod_languages/' . $language->image . '.gif', $params->get('full_name') ? '' : $language->title_native, null, true); ?>`

Should be something like that

`<?php echo HTMLHelper::_('image', 'mod_languages/' . $language->image . '.gif', $language->title_native, ['title' => $language->title_native, 'width' => '18', 'height' => '12'], true); ?>`

Occurences:

https://github.com/joomla/joomla-cms/blob/a3106be3f83b8793f175e59507e6fa21aaf1eb00/modules/mod_languages/tmpl/default.php#L35
https://github.com/joomla/joomla-cms/blob/a3106be3f83b8793f175e59507e6fa21aaf1eb00/modules/mod_languages/tmpl/default.php#L54
https://github.com/joomla/joomla-cms/blob/a3106be3f83b8793f175e59507e6fa21aaf1eb00/modules/mod_languages/tmpl/default.php#L64
https://github.com/joomla/joomla-cms/blob/a3106be3f83b8793f175e59507e6fa21aaf1eb00/modules/mod_languages/tmpl/default.php#L88
https://github.com/joomla/joomla-cms/blob/a3106be3f83b8793f175e59507e6fa21aaf1eb00/modules/mod_languages/tmpl/default.php#L103

Maybe that with the params and full name is fine on some cases as it includes it so then it will prevent the  Image elements do not have [alt] attributes that are redundant text. but pagespeed throws an error sometimes.

### Device and system info

* What version of Joomla were you using?
Joomla 6

* What device were you using? (eg, phone, laptop)
Mobile Device

* What web browser were you using? (eg, Chrome, Safari)
Firefox

* If you were using assistive technology, please let us know the device or software you were using (eg VoiceOver, JAWS, sip-and-puff device, etc)
Pagespeed Insights

### Additional comments

If you know which [WCAG](https://www.w3.org/TR/WCAG/) or [ATAG](https://www.w3.org/TR/ATAG20/) criteria this violates, please indicate it here:

**Comment by chaitali-tekdi** [comment]
Issue #46743 is pull request number

**Comment by chaitali-tekdi** [comment]
Sorry, this is 46792 pull number
